src/options.js

```diff
@@ -945,3 +955,9 @@ exports.removed = {
   smarttabs: true,
   trailing: true
 };
+
+// Add options here which should not be automatically enforced by
+// `enforceall`.
+exports.noenforceall = {
```

Comment:
This is obviously necessary, but it does make the "enforceall" option a little confusing ("enforcesome"?). What do you think about adding to the documentation of the `enforceall` option that (in addition to being deprecated) it only enforces the options that were available in the 2.6.3 release of JSHint?